UI/Navigation/src/Navigation/Presentation/ContentControlNavigation/CControlNavigationPage.xaml

```diff
@@ -0,0 +1,44 @@
+﻿<Page x:Class="Navigation.Presentation.CControlNavigationPage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:local="using:Navigation"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  xmlns:uen="using:Uno.Extensions.Navigation.UI"
+	  xmlns:utu="using:Uno.Toolkit.UI"
+	  Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+	  mc:Ignorable="d">
+
+	<Grid x:Name="ParentGrid">
+		<Grid.RowDefinitions>
+			<RowDefinition Height="Auto" />
+			<RowDefinition />
+		</Grid.RowDefinitions>
+		<utu:NavigationBar Padding="60,0,0,0" Content="Content Control Navigation" />
+
+		<Grid Grid.Row="1">
+			<Grid.ColumnDefinitions>
+				<ColumnDefinition Width="*" />
+				<ColumnDefinition Width="*" />
+			</Grid.ColumnDefinitions>
```

Comment:
On large screens this looks nice, but small ones it's not so pratical:

![image](https://github.com/user-attachments/assets/ca2d7fea-dd09-47a7-b50b-df0cd3a6cd79)

Maybe we could apply this: https://github.com/unoplatform/uno.extensions/blob/main/testing/TestHarness/TestHarness/Ext/Navigation/Apps/Commerce/CommerceProductsPage.xaml#L29-L36